modules/openapi-generator/src/main/resources/typescript-fetch/modelGeneric.mustache

```diff
@@ -10,14 +10,6 @@ import {
 {{/imports}}
 
 {{/hasImports}}
-{{#discriminator}}
-import {
-{{#discriminator.mappedModels}}
-     {{modelName}}FromJSONTyped{{^-last}},{{/-last}}
-{{/discriminator.mappedModels}}
-} from './';
-
-{{/discriminator}}
```

Comment:
I might be horribly wrong here, but this seems to be needed for `legacyDiscriminatorBehavior = true`, however, there is a bug here as well, as `modelName`s might not be unique. Unfortunately, I am not familiar with mustache and have no idea how to fix that

Reply:
You might be right on this one. If I check out @GeroSchaarmann 's branch, and set legacyDiscriminatorBehavior=true those imports are missing.
![import missing for legacyDiscriminatorBehavior = true ](https://github.com/OpenAPITools/openapi-generator/assets/126659284/956f616d-4e8b-4bfe-b31f-7a7620bfe9ba)